vita3k/gui/src/about_dialog.cpp

```diff
@@ -145,17 +145,17 @@ void draw_about_dialog(GuiState &gui, EmuEnvState &emuenv) {
         };
 
         // Developers list
-        for (const auto developer : developers_list)
+        for (const auto &developer : developers_list)
```

Comment:
there is no need to use reference to pointer. (it's not a common rule to always use reference in such loops)

Reply:
OK